packages/ember-glimmer/lib/resolver.ts

```diff
@@ -262,21 +262,16 @@ export default class RuntimeResolver implements IRuntimeResolver<OwnedTemplateMe
 
   private _lookupPartial(name: string, meta: OwnedTemplateMeta): PartialDefinition {
     const template = lookupPartial(name, meta.owner);
-    const partial = new PartialDefinition(name, lookupPartial(name, meta.owner));
 
     if (template) {
-      return partial;
+      return new PartialDefinition(name, template);
     } else {
       throw new Error(`${name} is not a partial`);
     }
   }
 
   private _lookupModifier(name: string) {
-    let modifier = this.builtInModifiers[name];
-    if (modifier !== undefined) {
```

Comment:
there is already `undefined` check downstream in `handle` method, so this looks unnecessary